src/routes/init/index.ts

```diff
@@ -1,20 +1,22 @@
 import { Request } from 'express';
 import { DbClient, IDbClient } from '../../dal/interface';
 import { InsertWriteOpResult } from 'mongodb';
-import { data } from './output';
+import { importDbEntries } from '../dbInit/import';
 import { applyTracing } from '../../utils/tracer/index';
 import { CSEResponse } from '../../server/index';
 
 
 export const handler = async (req: Request, res: CSEResponse): Promise<void> => {
+    const output: string = await importDbEntries();
     const dbArgs: IDbClient = {
         url: 'mongodb://mongo',
         port: 27017,
         dbName: 'cse'
     };
     const client: DbClient = new DbClient(dbArgs);
     await client.connect();
-    const result: InsertWriteOpResult | undefined = await client.insert(data);
+    // tslint:disable-next-line
+    const result: InsertWriteOpResult | undefined = await client.insert(JSON.parse(output));
```

Comment:
Is there a particular reason why `output` needs to be a string? Looking at the `importDbEntries` function, it seems like it would make more sense to return an object and then insert that, without having to `JSON.stringify/JSON.parse` the constructed entries.